subprojects/gdk_rust/tests/integration.rs

```diff
@@ -355,6 +355,52 @@ fn labels() {
     test_session.stop();
 }
 
+#[test]
+fn rbf() {
+    // Create session/account and fund id
+    let mut test_session = setup_session(false, 0, |_| ());
+    test_session
+        .session
+        .create_subaccount(CreateAccountOpt {
+            name: "Account 1".into(),
+            subaccount: 1,
+        })
+        .unwrap();
+    test_session.node_sendtoaddress(&test_session.get_receive_address(1).address, 9876543, None);
+    test_session.wait_tx_status_change();
+
+    // Create transaction for replacement
+    let mut create_opt = CreateTransaction::default();
+    let dest_address = test_session.get_receive_address(1).address;
+    create_opt.subaccount = 1;
+    create_opt.addressees.push(AddressAmount {
+        address: dest_address,
+        satoshi: 50000,
+        asset_id: None,
+    });
+    create_opt.fee_rate = Some(25000);
+    create_opt.memo = Some("poz qux".into());
+    let tx = test_session.session.create_transaction(&mut create_opt).unwrap();
+    let signed_tx = test_session.session.sign_transaction(&tx).unwrap();
+    let txid = test_session.session.broadcast_transaction(&signed_tx.hex).unwrap();
+    test_session.wait_account_tx(1, &txid);
+    let txitem = test_session.get_tx_from_list(1, &txid);
+    assert_eq!(txitem.fee_rate / 1000, 25);
+
+    // Replace it
+    let mut create_opt = CreateTransaction::default();
+    create_opt.subaccount = 1;
+    create_opt.previous_transaction = Some(txitem);
+    create_opt.fee_rate = Some(43000);
+    let tx = test_session.session.create_transaction(&mut create_opt).unwrap();
+    let signed_tx = test_session.session.sign_transaction(&tx).unwrap();
+    let txid = test_session.session.broadcast_transaction(&signed_tx.hex).unwrap();
+    test_session.wait_account_tx(1, &txid);
+    let txitem = test_session.get_tx_from_list(1, &txid);
```

Comment:
Is the replaced tx still in the tx list?
In both cases, we should add few lines to test the intended behavior

Reply:
To my understanding it should not be in the list, the same as how gdk-cpp behaves.

But there was a bug that prevented this from working, fixed in https://github.com/Blockstream/gdk/pull/119/commits/278ac0996ae879652aee29d9dbcb0f69d0c7e14b and added a test in https://github.com/Blockstream/gdk/pull/119/commits/dcd661c2c49235930c987018aa797337b6d3b24c.